lib/git_reflow/commands/setup.rb

```diff
@@ -15,11 +15,24 @@
     end
 
     choose do |menu|
-      menu.header = "Available remote Git Server services:"
+      menu.header = "Available remote Git Server services"
       menu.prompt = "Which service would you like to use for this project?  "
 
       menu.choice('GitHub')    { GitReflow::GitServer.connect reflow_options.merge({ provider: 'GitHub', silent: false }) }
       menu.choice('BitBucket (team-owned repos only)') { GitReflow::GitServer.connect reflow_options.merge({ provider: 'BitBucket', silent: false }) }
     end
+
+    choose do |lgtm_menu|
```

Comment:
since there is only a single choice, let's change this to an ask:

```ruby
GitReflow::Config.add "constants.minimum_approvals", ask("Set the minimum number of approvals (leaving blank will require approval from all commenters): ")
```